deploy/101_setup_sft.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2021 The Wolfpack
+ * This file is part of wolves.finance - https://github.com/wolvesofwallstreet/wolves.finance
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * See the file LICENSES/README.md for more information.
+ */
+
+/* eslint @typescript-eslint/no-var-requires: "off" */
+
+//const ethers = require('ethers');
+const fs = require('fs');
+
+require('hardhat-deploy');
+require('hardhat-deploy-ethers');
+
+// TODO: Fully qualified contract names
+const TOKEN_CONTRACT = 'WowsToken';
+const SFT_CONTRACT = 'WOWSERC1155';
+const SFT_MINTER_CONTRACT = 'WOWSSftMinter';
+
+// Path to generated address registry file
+const ADDRESS_REGISTRY = `${__dirname}/../src/config/generated-addresses.json`;
+
+// Helper function
+function log_step(step_string) {
+  console.log(`\n==> ${step_string}\n`);
+}
+
+/**
+ * Steps to deploy the WOWS SFT environment
+ */
+const func = async function (hardhat_re) {
+  const { deployments, getNamedAccounts } = hardhat_re;
+
+  const { execute } = deployments;
+  const { marketingWallet } = await getNamedAccounts();
+
+  // Load contract addresses
+  const addressRegistry = JSON.parse(
+    fs.readFileSync(ADDRESS_REGISTRY).toString()
+  );
+
+  const addresses = addressRegistry.hardhat;
+
+  const TOKEN_INSTANCE = await hardhat_re.ethers.getContract(TOKEN_CONTRACT);
+  const SFT_INSTANCE = await hardhat_re.ethers.getContract(SFT_CONTRACT);
+
+  //////////////////////////////////////////////////////////////////////////////
+  //
+  // MultiSig marketing wallet calls for SFT
+  //
+  //////////////////////////////////////////////////////////////////////////////
+
+  log_step('Marketing wallet calls for SFT');
+
+  //
+  // 1.) WowsToken:: grantRole (REWARD_ROLE, WOWSSftMinter.sol)
+  //
+
+  await execute(
```

Comment:
These calls will not succeed in mainnet due to multisig, just want to mention....

Reply:
that's the reason for splitting up deployment steps into "deploy" and "setup". We only run "deploy" on mainnet because of the multisig

Reply:
Ok, sounds great!